cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -86,6 +87,10 @@ func init() {
 	if err != nil {
 		glog.Fatalf("Failed to create client: %v", err)
 	}
+	snapClient, err := snapclientset.NewForConfig(config)
```

Comment:
What does `NewForConfig` mean? is it `NewSnapClientFromConfig`?

Reply:
Added a comment for this: "NewForConfig creates a new Clientset for VolumesnapshotV1alpha1Client"